GiEnJul/ClientApp/src/components/registerGiver/IGiverFormData.tsx

```diff
@@ -0,0 +1,9 @@
+interface IGiverFormData {
+  location: string;
+  fullname: string;
+  email: string;
+  phoneNumber: string;
+  maxReceivers: string;
```

Comment:
I checked into this and concluded that it's best to keep it as a string. This is because it's an select tag. When no item is selected the value is set to "". This could maybe be set to undefined, but I think this will make some problems with react (uncontrolled variable). Because of this empty string, I think that the value always should be a string.